src/main/java/com/easysplit/ess/user/domain/models/User.java

```diff
@@ -18,6 +18,10 @@ public class User {
     private Timestamp createdDate;
     private List<Link> links;
 
+    public User() {
```

Comment:
If a constructor is empty, would it not be better to keep its body at a minimum? example: public User() {}